src/config/lwwMap.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { entriesOf } from '@salesforce/ts-types';
+import { nowBigInt } from '../util/time';
+import { LWWRegister } from './lwwRegister';
+import { ConfigContents, Key } from './configStackTypes';
+
+export const SYMBOL_FOR_DELETED = 'UNIQUE_IDENTIFIER_FOR_DELETED' as const;
```

Comment:
I'm not sure there's actually any benefit to using this but it might be more idiomatic to use `Symbol('UNIQUE_IDENTIFIER_FOR_DELETED')`

Reply:
I tried that first (that's why it's still called SYMBOL...).  There's something in TS that distinguishes between Symbol and unique symbol. https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html

And the exported symbol didn't satisfy unique symbol.  And I couldn't figure out why, so I gave up and did this and moved on.  I shoulda left a note about that.

I'd *love* your TS-fu if you feel like attempting to make that work.

Reply:
FWIW, it works inside the file, but the export of it and import elsewhere is where it broke for me.